website/package.json

```diff
@@ -1,10 +1,12 @@
 {
   "scripts": {
-    "add-contributors-on-site": "cp ../CONTRIBUTORS.md ../docs/contributors.md",
-    "start": "yarn add-contributors-on-site && docusaurus start",
-    "build": "yarn add-contributors-on-site && docusaurus build",
+    "add-contributors-on-site": "node ../copyFile.js ../CONTRIBUTORS.md ../docs/contributors.md true",
+    "add-contributing-on-site": "node ../copyFile.js ../CONTRIBUTING.md ../docs/contributing.md true",
+    "add-pages-on-site": "yarn add-contributors-on-site && yarn add-contributing-on-site",
```

Comment:
honestly, i would rather you hardcode this src and dest info into `copyFile.js`. i don't like bloating package.json too much. this isn't a blocker, i'm just giving you feedback on how i would do things.  nice work!

Reply:
@sw-yx , thank you!

You're right, if you don't mind I will add it in separate pull-request to fix.